org.scala-ide.sdt.core/src/org/scalaide/util/eclipse/FileUtils.scala

```diff
@@ -98,11 +99,14 @@ object FileUtils {
    * file might "mounted" under multiple locations in the Eclipse file system, the `prefix`
    * path is used disambiguate.
    */
-  def resourceForPath(location: IPath, prefix: IPath = Path.EMPTY): Option[IFile] = {
-    val resources = Try(ResourcesPlugin.getWorkspace.getRoot.findFilesForLocationURI(URIUtil.toURI(location))).getOrElse(Array())
+  def fileResourceForPath(location: IPath, prefix: IPath = Path.EMPTY): Option[IFile] = {
```

Comment:
This changes actually the public API but I guess no one depends on this code anyway so we can just ignore it ;)